repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/pooling/ResourceContext.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.openrepose.commons.utils.pooling;
 
+@FunctionalInterface
 public interface ResourceContext<R, T> {
 
     T perform(R resource) throws ResourceContextException;
```

Comment:
Man are the names bad here, they give me no idea what the intended usage of this interface is.